navit/gui/gtk/gui_gtk_poi.c

```diff
@@ -146,6 +161,21 @@ model_poi (struct gtk_poi_search *search)
 					char direction[5];
 					gtk_list_store_append(search->store_poi, &iter);
 					get_compass_direction(direction,transform_get_angle_delta(&center,&coord_item,0),1);
+
+                                        /* If the user has selected
```

Comment:
This comment probably doesn't need to be broken down so much.
We should up the line limit to probably 120 or so?